pkg/controllers/cstorvolumeconfig/volume_operations.go

```diff
@@ -697,38 +733,49 @@ func (c *CVCController) getUpdatePDBForVolume(cvcObj *apis.CStorVolumeConfig) (s
 	if !isHAVolume(cvcObj) {
 		return "", nil
 	}
-	pdbObj, err := c.getOrCreatePodDisruptionBudget(getCSPC(cvcObj), cvcObj.Status.PoolInfo)
-	if err != nil {
-		return "", err
+
+	if useV1beta1PodDisruptionBudget {
+		pdbObj, err := c.getOrCreateV1beta1PodDisruptionBudget(getCSPC(cvcObj), cvcObj.Status.PoolInfo)
```

Comment:
nit: I personally don't like long named methods, if other PR reviewers feel the same way, maybe the names can be changed to `getOrCreatev1beta1PDB`.

Reply:
Reverted to the previous function name. Reverted changes which supported both policy/v1beta1 and policy/v1. Only support for policy/v1 will be there.